main_logic/tts_client.py

```
@@ -1098,6 +1098,230 @@ def gemini_tts_worker(request_queue, response_queue, audio_api_key, voice_id):
+                # 新的语音ID，清空缓冲区并重新开始
+                if current_speech_id != sid and sid is not None:
+                    current_speech_id = sid
+                    text_buffer = []
+                
+                if sid is None:
+                    # 收到终止信号，合成累积的文本
+                    if text_buffer and current_speech_id is not None:
+                        full_text = "".join(text_buffer)
+                        if full_text.strip():
+                            try:
+                                # 构建请求参数（使用 api_v2 的 /tts 端点）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)